todo_app/data/trello_items.py

```diff
@@ -1,24 +1,40 @@
 import requests
 import json
+import os
 from todo_app.data.item import Item
 from todo_app.data.trelloList import TrelloList
 import todo_app.data.trello_constants as constants
+import iso8601
+import vcr
 
 class Trello_service(object):
     trello_lists = {}
-
-    def __init__(self):
-        self.get_lists()
-        self.get_items_from_trello()
+    def get_auth_params(self):
+        return { 'key': os.getenv('TRELLO_KEY'), 
+                'token': os.getenv('TRELLO_TOKEN'),
+                'list': os.getenv('TRELLO_BOARD_ID'),
+                'useVcr': os.getenv('USE_VCR')}
```

Comment:
Your usage of VCR seems messy: you've modified your application code to check whether tests are running and if so, behave totally differently.

This file shouldn't need to be aware of VCR. You can just patch `requests` to use cassettes as appropriate.

Reply:
Removed VCR